Tests/TestingTests/Traits/CommentTests.swift

```diff
@@ -57,6 +57,15 @@ struct CommentTests {
   func explicitlyNilComment() {
     #expect(true as Bool, nil as Comment?)
   }
+
+  @Test("String interpolation")
+  func stringInterpolation() {
+    let value1: Int? = 123
+    let value2: Int? = nil
+    let value3: Any.Type = Int.self
+    let comment: Comment = "abc\(value1)def\(value2)ghi\(value3)"
```

Comment:
I'd include some type which explicitly conforms to `CustomTestStringConvertible`, and customizes the value of its `testDescription` property to differ from `description`.

Reply:
`Int?` conforms to `CustomTestStringConvertible`.

Reply:
Ok, that's somewhat subtle but I see.

Reply:
I can add additional testing with a local type too.